api/v1alpha1/policy_setting.go

```diff
@@ -16,24 +16,17 @@ limitations under the License.
 
 package v1alpha1
 
-import (
-	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
-)
-
-type PolicySettingName string
+import v1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
```

Comment:
Putting some comments from the earlier (initial) commit:

**API:**
action_behaviour
	recommend and external_approval modes could be added.
policy_setting
	PolicySettingNames here are SLO specific, whereas PolicySetting structure seems very generic.

**Samples:** 
You could add comments to the sample yaml files (copy from api definitions perhaps)

Reply:
To be honest I also find the `SLO` in the policy name a little odd, but I haven't been able to think of a better suggestion.

Reply:
@pankajbatra1 , I added `Recommend` to the action mode. I don't think external approval will be needed anytime soon, and definitely won't be needed for SLO driven actions (this has been confirmed with @evat-pm a long time ago). We can add this mode in the future if we see a need, and it will not break backward compatibility.

`PolicySettingName` type has been removed.

Comments have been added to sample yamls.